theme/layouts/_navigation.ejs

```diff
@@ -1,4 +1,32 @@
-<nav id="main-navigation">
+<% const headerMenuItems = ["documentation", "tutorial", "ready-projects", "what's-new"];
+const regularExpression = /\\|\//;
+const flyoutPages = site.queryPages(({originalPathname}) => headerMenuItems.includes(originalPathname.split(regularExpression)[0]));
+
+let navigation = [];
+let children = [];
+let obj = {};
+let value = {};
+
+for (const {originalPathname, pathname, title} of flyoutPages) {
+  console.log(originalPathname.split('\\')[1], originalPathname.split('\\')[2])
```

Comment:
CMintS is currently [broken on windows](https://github.com/cmints/cmints/issues/195), I think CMintS needs to be first fixed before proceeding with the navigation, as it requires the paths to be correct.